src/formatDate.js

```diff
@@ -8,7 +8,38 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const sepatareDate = date.split(fromFormat[3]);
```

Comment:
There is a typo in the variable name `sepatareDate`. It should be `separateDate` to correctly reflect its purpose.

Reply:
i have corrected